templates/chain.html

```diff
@@ -257,32 +256,33 @@ <h4>Manage operations</h4>
               </div>
               <br>
               <select id="operations" style="margin-bottom: 15px;" onchange="refresh()">
-                <option value="" disabled selected>Select an operation</option>
+                <option value="" disabled selected>Operations</option>
                 {% for op in operations %}
                   <option id="chosen-operation" value="{{ op.id }}">{{ op.name }} - {{ op.start }}</option>
                 {% endfor %}
               </select>
-              <input id="queueName" style="opacity:0.5" disabled="true" placeholder="Enter a name"/>
-              <select name="work" id="queueFlow" style="opacity:0.5" disabled="true">
-                <option value="" disabled selected>Select an adversary</option>
+              <input id="queueName" class="queueOption" style="opacity:0.5" disabled="true" placeholder="Operation name"/>
+              <select name="work" id="queueFlow" class="queueOption" style="opacity:0.5" disabled="true">
+                <option value="" disabled selected>Adversary</option>
                 {% for adv in adversaries %}
                   <option id="qflow-{{ adv }}" value="{{ adv.id }}">{{ adv.name }}</option>
                 {% endfor %}
               </select>
-              <select name="host" id="queueGroup" style="opacity:0.5" disabled="true">
-                <option value="" disabled selected>Select a group</option>
+              <select name="host" id="queueGroup" class="queueOption" style="opacity:0.5" disabled="true">
+                <option value="" disabled selected>Group</option>
                 {% for g in groups %}
                   <option id="qgroup-{{ g.name }}" value="{{ g.id }}">{{ g.name }}</option>
                 {% endfor %}
               </select>
-              <select name="cleanup" id="queueCleanup" style="opacity:0.5" disabled="true">
+              <select name="cleanup" id="queueCleanup" class="queueOption" style="opacity:0.5" disabled="true">
                 <option value="1" selected>Clean up artifacts</option>
-                <option value="0">Turn off cleanup</option>
+                <option value="0">Leave Artifacts</option>
               </select>
-              <select name="stealth" id="queueStealth" style="opacity:0.5" disabled="true">
+              <select name="stealth" id="queueStealth" class="queueOption" style="opacity:0.5" disabled="true">
                 <option value="0" selected>Run normal operation</option>
-                <option value="1">Run stealthy operation</option>
+                <option value="1">Run stealth operation</option>
               </select>
+              <input name="jitter" id="queueJitter" class="queueOption" placeholder="Jitter (min/max)" style="opacity:0.5;" disabled="true"/>
```

Comment:
two things: 1) validation of this being a fraction < 1 and 2) some explanation of what a jitter fraction is